BinanceExchange.API/Endpoints.cs

```diff
@@ -219,6 +220,9 @@ private static string GenerateQueryStringFromData(IRequest request)
             {
                 throw new Exception("No request data provided - query string can't be created");
             }
+
+            _settings.Converters.Add(new StringDecimalConverter());
```

Comment:
If you could instead use this as an attribute like https://github.com/glitch100/BinanceDotNet/blob/master/BinanceExchange.API/Models/Request/CreateOrderRequest.cs#L19 , then we can get this merged. Reason being it provides a performance benefit and given people require speed with this wrapper, we are focusing on that.